pendulum_control/src/pendulum_demo.cpp

```diff
@@ -129,12 +130,15 @@ int main(int argc, char * argv[])
   // message pool is determined by the number of threads (the maximum number of concurrent accesses
   // to the subscription).
   // Since this example is single-threaded, we choose a message pool size of 1 for each strategy.
+  const size_t messagePoolSize = 32;
```

Comment:
We still need that in our case we use this size?

Reply:
Done